gateway/src/node/mod.rs

```diff
@@ -636,6 +660,13 @@ impl<St> Gateway<St> {
             shutdown.fork("mixnet_handling::Listener"),
         );
 
+        let shared_session_stats = self.session_stats.take().unwrap_or_default();
```

Comment:
Maybe just skip starting it if it has not been set? Or maybe it is assumed to always be running?

Reply:
I got a bit baited by that similar construction in [`mixing_stats`](https://github.com/nymtech/nym/blob/08aa0af56242bdc2fcf46f5e07b05557855f61e5/mixnode/src/node/mod.rs#L133).
If we have to create a default one here it won't be passed to the self described API so it's not really useful to start it.

Skipping start in 65b83627fc31506ef8725adf0cd21632f68a55aa